File: dotnet/NppCore/Models/PlayerMatches.cs
public class PlayerMatches
{
    public Guid PlayerId { get; set; }
    public string Year { get; set; }=string.Empty;
    public DateTimeOffset Match_time { get; set; }
    public Guid OpponentId { get; set; }
    public string OpponentUsername { get; set; }=string.Empty ;
    public string Score { get; set; }=string.Empty;
    public string Result { get; set; }=string.Empty;

}

File: react/src/types/playerMatches.ts
export interface PlayerMatches
{
    playerId:string;
    opponentId:string;
    opponentUsername:string;
    score:string;
    result:string;
    year:string;
    match_time:string;
}

export interface PlayerMatchesRequest
{
    opponentId:string;
    opponentUsername:string;
    score:string;
    result:string;
    year:string;
    match_time:string;
}

export interface PlayerMatchesResponse
{
    playerId:string;
    opponentUsername:string;
    result:string;
    score:string;
    match_time:string;
}